index.js

```diff
@@ -21,11 +21,11 @@ exports = module.exports = (function () {
     var lazyLoad = function () {
         return getHarness().apply(this, arguments);
     };
-    
```

Comment:
Please revert these unrelated whitespace changes; that's part of the style of this project.

Reply:
@ljharb done

Reply:
@ljharb although, I'm curious -- why would you keep trailing whitespaces, such as that in https://github.com/substack/tape/blob/7261ccc40b207040f0b2586775ca12164ae93922/lib/test.js#L147 for style?

Reply:
It's not my decision; it's the style of the repo owner.